db/seeds.rb

```diff
@@ -6,3 +6,4 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
```

Comment:
You should delete this lines :wink:

Reply:
I only left the first part that explains the functionality of seeds.rb